tests/specs/config_changes/CircularJsrDep02.txt

```diff
@@ -5,7 +5,7 @@
     "specifiers": {
       "jsr:@scope/package_a": "jsr:@scope/package_a@0.0.1",
       "jsr:@scope/package_b": "jsr:@scope/package_b@0.0.1",
-      "jsr:@scope/package_c": "jsr:@scope/package_b@0.0.1"
+      "jsr:@scope/package_c": "jsr:@scope/package_c@0.0.1"
```

Comment:
This was previously passing because of the missing condition in src/lib.rs (this test was setup wrong)